conductor_api/src/config.rs

```diff
@@ -382,6 +385,24 @@ pub struct DnaConfiguration {
     pub hash: Option<String>,
 }
 
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
+pub enum ISOActive {
+    On,
+    Off,
+}
```

Comment:
Hmm why not just a boolean for this? Do you anticipate other variants?

Reply:
Anticipating a Fixed Time variant which the can specify which UTC to use